app/src/main/java/de/westnordost/streetcomplete/controls/TeamModeColorCircleView.kt

```diff
@@ -0,0 +1,54 @@
+package de.westnordost.streetcomplete.controls
+
+import android.content.Context
+import android.graphics.Color
+import android.util.AttributeSet
+import androidx.annotation.ColorInt
+import androidx.constraintlayout.widget.ConstraintLayout
+import de.westnordost.streetcomplete.R
+import kotlinx.android.synthetic.main.view_team_mode_color_circle.view.*
+
+class TeamModeColorCircleView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : ConstraintLayout(context, attrs, defStyleAttr) {
+
+    fun setTeamIndex(index: Int) {
```

Comment:
Shouldn't this be `setIndexInTeam` again to avoid confusion?

Reply:
Hm, right

Reply:
Fixed in #2613.